client/src/pages/Footer.css

```diff
@@ -0,0 +1,4 @@
+.main-footer {
+    background-color: dimgray;
+    color: aliceblue;
+}
```

Comment:
This is great, one last request is to put the Footer.css file in the components folder next to Footer.js